### PR TITLE
My Portfolio

### PR DESCRIPTION
**Pull request by eric2umeh**
- completed the web version of the portfolio
- checked for linter errors

### CODE REVIEW

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @eric2umeh 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Professional README file ✔️

## Required Changes:

- [x] Kindly add the MIT file to your project, the link in the README file is broken 😓 
- [x] Kindly try to follow best  practices when you write commit messages, they should be capitalized sentences in the imperative form, example: "Add linters" instead of "added linters" 👍 

NOTE: there isn't alot of the code in the changed files so I'll leave the comments here instead of inline comments 👍 

- [x] Kindly add `font-family` to your texts to match the design in Figma 👍 

### Your design:
image

### Figma design:
image

- [x] Kindly match the navbar of the Figma design, yours is lacking some space on the top, the logo should be bolder and the yellow underline in `portfolio` should not be there 👍 

### Your design:
image

### Figma design:
image

- [x] Kindly increase the size of the project images, in the cards where the image is on the right the text should be aligned to the left instead of to the right. Kindly fix these issues 💯 

### Your design:
image

### Figma design:
image

- [x] Kindly add some space between the left part and the right part to improve visibility and to match the design in Figma 👍 

### Your design:
image

### Figma design:
image

- [x] Kindly reduce the `width` of the little languages cards, and please add some `padding-bottom` to it to match the design in Figma 👍 

### Your design:
image

### Figma design:
image

- [x] Kindly add the missing arrow icon at the end of this part 👍 

### Your design: 
image

### Figma design:
image

- [x] Kindly reduce the size of the input fields to match the design in Figma 👍 

### Your design:
image

### Figma design:
image




------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -84,7 +72,7 @@ To get more details, click om the available links
 
 👤 **Eric**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
+- GitHub: [@eric2umeh](https://github.com/githubhandle)
 - Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
Comment (pinkmoon25) [inline README.md:76]: - [x] As requested by the previous reviewer, please fix the MIT link in your readme as currently it is giving a 404 error, you have coorectly added the MIT.md file but you have not changed the link here :point_right: `This project is [MIT](./LICENSE) licensed.` change the link to be` ./MIT.md`

File: index.html [hunk 2]
```diff
@@ -98,7 +97,7 @@ <h2 class="work-title">Tonic</h2>
           <article class="work" id="work-img-2">
             <img
               class="work-img-2"
-              src="assets/bg-mobile-2.svg"
+              src="assets/work2.svg"
               alt="First Project"
             />
             <div class="item2">
```
Comment (pinkmoon25) [inline index.html:103]: - [x] As requested by the previous reviewer please make sure our project cards follow the figma template your design :point_down:  the buttons sre not in the right place
image
 
figma design :point_down: 
image

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @eric2umeh , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
linters checks :heavy_check_mark: 
follows figma design :heavy_check_mark: 
used flex and grid :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @eric2umeh 

Your project is complete! Great job for making the requested changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request✔️ 
- Responsive site✔️ 
- Design well followed ✔️ 
- No linter errors✔️ 

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**